docs/vm-installation.md

```diff
@@ -0,0 +1,99 @@
+# Installing antrea-agent on VM
+
+`antrea-agent` can be run on Linux/Windows VMs. Below are the steps needed to configure and run
+antrea-agent on VMs
+
+## Prerequisites on Kubernetes cluster
+
+1. Create a Namespace for `antrea-agent`
+2. Create a ServiceAccount, ClusterRole and ClusterRoleBinding for `antrea-agent`.
```

Comment:
Besides these two, maybe we also need to mention that users need to make sure ExternalNode feature switch is enabled for Antrea controller?

Reply:
Yes, I will add it as a pre-requisite.